### ISSUE TITLE
stat options not working in 2.1.0-beta.11

### ISSUE THREAD

**Issue by cortopy** [webpack/webpack-dev-server#697]
**I'm submitting a bug report**

**webpack and webpack-dev-server version:**
webpack: 2.1.0-beta.26
webpack-dev-server: 12.1.0-beta.11

**Please tell us about your environment:**
Linux
Running server via Node.js API

**Current behavior:**
I have the following in my config

```javascript
devServer: {
    quiet: true,
    stats : {
      hash        : false,
      version     : false,
      timings     : false,
      assets      : false,
      chunks      : false,
      modules     : false,
      reasons     : false,
      children    : false,
      source      : false,
      errors      : false,
      errorDetails: false,
      warnings    : false,
      publicPath  : false,
    }
}
```

And despite having turned all options warning still output to console in browser. This is annoying because linter warnings would stop the reload.

The terminal output is as it should though

**Expected/desired behavior:**
No output to console in browser with this config. Same as with terminal

**Comment by SpaceK33z** [comment]
To disable browser logs you'll need to use `clientLogLevel: "none"`. This might seem a little redundant, but many people want only browser logs and no CLI logs.

Sidenote: `clientLogLevel: "none"` is broken at the moment in beta.11 (see #693), will be fixed in next beta.

**Comment by cortopy** [comment]
Having looked at the commit, I've made the same change  locally so that I can at least use it. `clientLogLevel` works but only for WDS messages. All the warnings are still output to the browser console though. 

The annoying part is that this new behaviour stops all reloads for me (warnings didn't stop earlier versions from reloading so never really bothered)

**Comment by SpaceK33z** [comment]
> All the warnings are still output to the browser console though.

Yeah true. Currently that is done on purpose, but perhaps I should change that behavior a bit so that when you use `"none"` it stops *all* logs from appearing. I assumed that nobody would *not* want to see error / warning logs in the devtools...

**Comment by cortopy** [comment]
Thanks @SpaceK33z  for amazingly fast response!!

Actually I would not normally want the warnings to disappear but not having a reload defeats the whole purpose of dev server. For what I gather it makes sense not to reload with errors, but my warnings are all about innocuous linting issues. In my case it would be better to reload with warnings like in older versions. Has this actually changed?

**Comment by SpaceK33z** [comment]
Ahh okay, you're right. It should reload for warnings. I'll fix that soon. Is your issue fixed after that?

**Comment by cortopy** [comment]
Yes!! Thank you so much!! I wouldn't expect dev server to sort my warnings too!! :joy:

**Comment by SpaceK33z** [comment]
Fix released in v2.1.0-beta.12.
